### PR TITLE
Wrap FLASH version 1.2.11

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: tools/flash/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Assembly
```
Comment (yhoogstrate) [inline tools/flash/.shed.yml]: `+ - Fastq Manipulation` ?

File: tools/flash/flash.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<tool id="flash" name="FLASH" version="1.2.11">
+    <description>adjust length of short reads</description>
+    <requirements>
+        <requirement type="package" version="1.2.11">flash</requirement>
+    </requirements>
+    <command detect_errors="aggressive">
```
Comment (yhoogstrate) [inline tools/flash/flash.xml]:
```
<version_command><![CDATA[
    flash --version | head -n 1
]]></version_command>
```

File: tools/flash/flash.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        <![CDATA[
+            flash --threads=\${GALAXY_SLOTS:-1}
+                -m $min_overlap
+                -M $max_overlap
+                -x $max_mismatch_density
+                $allow_outies
+                "$forward" "$reverse"
```
Comment (shiltemann) [inline tools/flash/flash.xml]: single quotes

File: tools/flash/flash.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        <param name="allow_outies" argument="--allow-outies" type="boolean" truevalue="--allow-outies" falsevalue="" checked="false" label="Combine read pairs in both orientations" help="FLASH uses the same parameters when trying each orientation. If a read pair can be combined in either orientation, the better-fitting one will be chosen using the same scoring algorithm that FLASH normally uses." />
+    </inputs>
+    <outputs>
+        <data format="fastq" name="merged_reads" from_work_dir="out.extendedFrags.fastq" label="${tool.name} on ${on_string}: Merged reads" />
+        <data format="fastq" name="unmerged_reads_f" from_work_dir="out.notCombined_1.fastq" label="${tool.name} on ${on_string}: Unmerged forward reads" />
+        <data format="fastq" name="unmerged_reads_r" from_work_dir="out.notCombined_2.fastq" label="${tool.name} on ${on_string}: Unmerged reverse reads" />
+        <data format="tabular" name="hist" from_work_dir="out.hist" label="${tool.name} on ${on_string}: Unmerged reads tabular histogram" />
+        <data format="text" name="histogram" from_work_dir="out.histogram" label="${tool.name} on ${on_string}: Unmerged reads histogram" />
```
[the remark this answers is not in the document]
Comment (davebx) [reply tools/flash/flash.xml]: You're probably right.

**Comment by bgruening** [comment]
Thanks @davebx!

**Pull request merged**